extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -206,6 +206,26 @@ $(function () {
 			return result;
 		}
 
+		function trackSave() {
+			Wikia.Tracker.track({
```

Comment:
wikia.tracker is and AMD module, you can require it instead calling one from window.